### DISCUSSION TITLE
[Feature] < Add option to change account/server in mobile app>

### DISCUSSION THREAD

**Discussion by ghost** [immich-app/immich#3131] [Feature Request]
### The feature

The ios mobile application has an unpleasant feature. If, for example, I changed the server port or url, then in the mobile application there is no way to change the account / server - since the application is constantly trying to connect to the previous saved account / server. At the moment there is no option to reset the application or just change the url adress of server while trying to connect to the old server

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by aureateflux** [comment]
I don't disagree that it makes sense to be able to make this change in settings, but does the iOS app not give you the option to change the endpoint when you log out? The Android app allows you to change the server endpoint url when you log out.

**Comment by bo0tzz** [comment]
Duplicate of #2335
